### ISSUE TITLE
PSR12.Operators.OperatorSpacing: false positives

### ISSUE THREAD

**Issue by jrfnl** [squizlabs/PHP_CodeSniffer#2696]
Looks like PR #2640 has introduced some new issues in `PSR12.Operators.OperatorSpacing` which extends `Squiz.WhiteSpace.OperatorSpacing`.

Example code:
```php
if (strpos($tokenContent, 'b"') === 0 && substr($tokenContent, -1) === '"') {}

$oldConstructorPos   = -1;

$content = -$content;
```

Previously, no errors would be thrown, now you get the following false positives:

```
-------------------------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
-------------------------------------------------------------------------------------------------------------
 3 | ERROR | [x] Expected at least 1 space after "-"; 0 found
   |       |     (PSR12.Operators.OperatorSpacing.NoSpaceAfter)
 5 | ERROR | [x] Expected at least 1 space after "-"; 0 found
   |       |     (PSR12.Operators.OperatorSpacing.NoSpaceAfter)
 7 | ERROR | [x] Expected at least 1 space after "-"; 0 found
   |       |     (PSR12.Operators.OperatorSpacing.NoSpaceAfter)
-------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------------------- 
```
